packages/cli/src/checks/__tests__/EXTERNAL_MISMATCH.ts

```diff
@@ -1,4 +1,4 @@
-import internalMismatch from "../EXTERNAL_MISMATCH";
```

Comment:
Interesting to see the naming discrepancy. Increases the diff, which might be problematic if we did end up sticking to a fork if the contribution to the main lib isn't accepted.